version.php

```diff
@@ -1,4 +1,18 @@
 <?php
+/**
+ * Software version identification.
+ *
+ * @link http://open-emr.org/wiki
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @author Robert Down <robertdown@live.com>
+ */
+
+require_once __DIR__ . "/vendor/autoload.php";
```

Comment:
Also, this setting OPENEMR__ENVIRONMENT would make much more sense as a global, wouldn't it?
I would hate to go back to the days of having settings in the files. Note having it in files also forces this for every instance in the multisite module when those should be per site(ie. another reason why globals is so nice).